<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>123</width>
      <height>184</height>
     </rect>
    </property>
    <property name="title">
     <string>Object</string>
    </property>
    <widget class="QPushButton" name="m_color">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>58</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QComboBox" name="m_objectChoice">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>25</y>
       <width>77</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Teapot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sphere</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cube</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="s_scaleLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>35</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="m_wireFrame">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>150</y>
       <width>97</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wireframe</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="m_scale">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>63</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="s_GLFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>101</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotate</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QSlider" name="m_rotX">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>25</y>
       <width>17</width>
       <height>90</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="m_rotY">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>25</y>
       <width>17</width>
       <height>90</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="m_rotZ">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>25</y>
       <width>17</width>
       <height>90</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
